src/modules/logicModule/abstracts/oracleIntegrations/UMA_OptimisticOracleV3/OptimisticOracleIntegrator.sol

```diff
@@ -233,17 +233,17 @@ abstract contract OptimisticOracleIntegrator is
 
         DataAssertion memory dataAssertion = assertionData[assertionId];
 
-        emit DataAssertionResolved(
-            assertedTruthfully,
-            dataAssertion.dataId,
-            dataAssertion.data,
-            dataAssertion.asserter,
-            assertionId
-        );
-
-        // If the assertion was true, then the data assertion is resolved.
+        // If the assertion was true, then the data assertion is resolved and an event is emitted.
         if (assertedTruthfully) {
             assertionData[assertionId].resolved = true;
+
+            emit DataAssertionResolved(
```

Comment:
hm
should we have an event then if its not asserted?

Reply:
Upon further discussion, this finding is invalid. I'll update the linear issue